src/main/java/com/weirddev/testme/intellij/ui/customizedialog/CustomizeTestDialog.java

```diff
@@ -155,19 +155,29 @@ protected void doOKAction() {
     }
 
     private boolean isMockable(PsiField psiField, PsiClass testedClass, String templateFileName) {
+        boolean isPrimitive = psiField.getType() instanceof PsiPrimitiveType;
```

Comment:
BTW, would be nice someday to move to all the logic dealing with PSI SDK  (`initMethodsTable()`, `initFieldsTable()`) to a new class and keep _CustomizeTestDialog_ focused on UI/user interaction